api4/user.go

```diff
@@ -2332,3 +2333,36 @@ func publishUserTyping(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func convertUserToBot(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireUserId()
+	if c.Err != nil {
+		return
+	}
+
+	user, err := c.App.GetUser(c.Params.UserId)
+	if err != nil {
+		c.Err = err
+		return
+	}
+
+	auditRec := c.MakeAuditRecord("convertUserToBot", audit.Fail)
```

Comment:
Why above did you put the audit code below he permission check and here you put it above?

Reply:
Aren't they both before the permission check?

Reply:
Yeah nevermind I mix up